src/connection/connection.js

```diff
@@ -133,21 +133,22 @@ class Connection {
 
     /**
      * User provided tags
+     * @type {string[]}
      */
     this.tags = []
   }
 
   /**
    * Get connection metadata
-   * @return {Object}
+   * @this {Connection}
```

Comment:
Should we document the object returned here?
`{ direction: string, ... }`

Reply:
Since I've hinted context, it picked up those things automatically:
https://github.com/libp2p/js-libp2p-interfaces/pull/64/files#diff-3ca9b26cdc518eaea397963f76e33cd5R102

Reply:
ok, cool! sounds good!